app/metrics.yaml

```diff
@@ -5302,3 +5302,18 @@ set_default_newtab_experiment:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2021-10-01"
+
+set_default_setting_experiment:
```

Comment:
Note it is not a misspelling but rather a choice due to the limitation of 29 characters.

Reply:
That limitation caused me a lot of headache before

Reply:
Same here! 😅